lib/tasks/assemble.js

```diff
@@ -65,9 +65,9 @@ class AssembleTask extends Task {
 
   pruneCommand() {
     if (shouldUseYarn(this.project.root)) {
-      return 'yarn install --production --no-bin-links';
```

Comment:
Do you remember why this was necessary before? I am guessing that it had to do with rebuilding native addons for electron.

Reply:
I don't recall, but this whole task is deleted in a later commit, so probably not worth worrying about.